packages/features/ee/dsync/page/team-dsync-view.tsx

```diff
@@ -39,6 +38,19 @@ const DirectorySync = () => {
     <div className="bg-default w-full sm:mx-0 xl:mt-0">
       <Meta title={t("directory_sync")} description={t("directory_sync_description")} />
       {HOSTED_CAL_FEATURES && <ConfigureDirectorySync orgId={currentOrg?.id || null} />}
+      {/* <SettingsToggle
+        toggleSwitchAtTheEnd={true}
+        title="Map groups to teams 1:1"
+        description="Members will be auto assigned to teams with the same name as their group."
+        switchContainerClassName="mt-6"
+      />
+      <SettingsToggle
+        toggleSwitchAtTheEnd={true}
+        title="Create new teams based on groups"
+        description="Automatically create new teams if a new group is pushed"
+        switchContainerClassName="mt-6"
+      />
+      <Button>Default team config</Button> */}
```

Comment:
Could also introduce these features later. For now, let's get this shipped.

Reply:
I'll come across this commented out code in 6 months and wonder about the context here, please create a follow up issue / maybe add that as a TODO comment.

Reply:
+1 on a follow up issue instead.